### ISSUE TITLE
Deeplab V3+

### ISSUE THREAD

**Issue by dannyhung1128** [kazuto1011/deeplab-pytorch#12]
Hi,
Can I simply change the model you used in ```train.py``` in to deeplab v3+?
I was wondering why you didn't do so, should it be quite straight forward?

Thanks!

**Comment by kazuto1011** [comment]
I would say you can run the code but it's not appropriate for training the v3+ model.
You need to change parameters to observe (line 30 -- 49 in train.py and optimizer setting) and other hyperparameters such as #batch and #output_stride, to train batch normalization parameters in the added modules on top of the dilated ResNet.
I've not implemented the v3+ training protocol yet.
